### ISSUE TITLE
Feature Request: Raise Error when using queue without starting it

### ISSUE THREAD

**Issue by peterHeuz** [tensorflow/tensorflow#18045]
When using functions like `tf.train.batch()` or `tf.train.shuffle_batch()` and probably more,  one needs to start the queue with `tf.train.start_queue_runners(sess)`. If used without it, above functions are still executable but hang indefinitely. I suggest raising an error if the queue hasn't been started before, as this has costed me a lot of time to figure out.

Alternatively point more directly to this usage in the documentation of the [functions](https://www.tensorflow.org/api_docs/python/tf/train/shuffle_batch) and the [guide](https://www.tensorflow.org/api_guides/python/reading_data#batching).

This might help a lot of people!

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tatianashp** [comment]
Thanks for reporting the issue. Please feel free to submit the PR that updates the documentation as you suggest.
